src/main/java/com/team4/testingsystem/utils/jwt/JwtTokenUtil.java

```diff
@@ -27,6 +27,10 @@
         }
     }
 
+    public long extractUserId(String token) throws JwtException {
+        return Long.parseLong(extractClaim(token, Claims::getId));
```

Comment:
Do we actually store user id in the claims? And why do you need extract it that way?